core/modules/user/user.module

```diff
@@ -2793,12 +2800,14 @@ function user_filters() {
   }
 
   $options = array();
+  $module_info = system_get_info('module');
   foreach (module_implements('permission') as $module) {
     $function = $module . '_permission';
+    $module_group = t('@module module', array('@module' => $module_info[$module]['name']));
```

Comment:
Perfect! (This is the same change the MR for Drupal 7 should include.) 👍🏻

Reply:
Indeed 👍🏼 ...naming the variable `$module_group` will make it easier to understand what's going on as well. I was confused by it before by merely reading the code and trying to understand what's happening - I had to spin up  test D7 site and see.